backend/composer/api/serializers.py

```diff
@@ -514,10 +528,11 @@ def create_statement_preview(self, instance, journey):
             statement = f"In {sex} {species}, the {phenotype} connection goes {journey_sentence}.\n"
         else:
             statement = f"A {phenotype} connection goes {journey_sentence}.\n"
+        
         statement += f"This "
-        if projection != "not specified":
+        if projection != "":
```

Comment:
@D-GopalKrishna imo we should use Python None instead of empty string

Reply:
Shifted it to Python None.